src/main/java/com/snowflake/kafka/connector/internal/telemetry/TelemetryConstants.java

```diff
@@ -9,21 +9,21 @@ public final class TelemetryConstants {
   public static final String STAGE_NAME = "stage_name";
   public static final String PIPE_NAME = "pipe_name";
 
-  public static final String PROCESSED_OFFSET = "processed_offset";
-  public static final String FLUSHED_OFFSET = "flushed_offset";
-  public static final String COMMITTED_OFFSET = "committed_offset";
-  public static final String PURGED_OFFSET = "purged_offset";
+  public static final String PROCESSED_OFFSET = "processed-offset";
```

Comment:
these were not used previously, changed to match the metricsUtil properties as we were accidentally using those previously